code/modules/research/designs.dm

```diff
@@ -1387,13 +1395,44 @@ datum/design/item/weapon/phoronpistol
 	materials = list("$metal" = 5000, "$glass" = 1000, "$phoron" = 3000)
 	build_path = /obj/item/weapon/gun/energy/toxgun
 
+datum/design/item/weapon/eglaive
+	id = "eglaive"
+	name = "energy glaive"
+	desc = "A Li'idra designed hardlight glaive reverse-engineered from schematics found amongst raider wreckages."
+	req_tech = list("combat" = 6, "phorontech" = 4, "materials" = 7, "syndicate" = 4,"powerstorage" = 4)
+	materials = list("$metal" = 10000, "$glass" = 18750, "$phoron" = 3000, "$silver" = 7500, "$diamond" = 11250)

```

Comment:
I'd say you're good to lower the amount of diamond for this. The mere presence of it on the list is a limitation enough, along with syndicate tech. With that high of a syndicate level, you could actually just remove diamond and we should be fine.